YoutubeKit/Common/Result.swift

```diff
@@ -10,4 +10,21 @@ import Foundation
 public enum Result<T, E> {
     case success(T)
     case failed(E)
+    public var value: T? {
```

Comment:
Is this diff need for this PR?

Reply:
no, this is not needed for this PR. I added this because instead of using `switch statement` I want to use syntax like this:
```
ApiSession.shared.send(request, closure: { (result) in
        if let value = result.value {
          print(value)
          
        } else if let error = result.error {
          print(error)
          
        }
        
      })
```
un...sorry I am new to open source project, next time I will try to see if I can separate things more clearly in commits. 😅

Reply:
This property is convenience, but a pull request should be minimum changes or separate if needed for review costs. (but it's ok is this time 👍)